Assets/MixedRealityToolkit/_Core/Devices/OpenVR/OpenVRDeviceManager.cs

```diff
@@ -179,35 +181,71 @@ private GenericOpenVRController GetOrAddController(XRNodeState xrNodeState)
                     break;
             }
 
-            IMixedRealityInputSource inputSource = InputSystem?.RequestNewGenericInputSource($"{CurrentControllerType} Controller {controllingHand}");
-
-            GenericOpenVRController detectedController = null;
+            Type controllerType;
 
-            // Initialize the controller base on the detected type
             switch (CurrentControllerType)
             {
                 case SupportedControllerType.GenericOpenVR:
-                    detectedController = new GenericOpenVRController(TrackingState.NotTracked, controllingHand, inputSource);
```

Comment:
why have you changed this, you have now broken the specific device implementation, meaning there is only one controller.
As discussed previously, we need the separate definitions to allow for variations between the controllers as required, input mappings, SDK models (when we support them) and others.

Please revert this back

Reply:
Will do. I thought I was being clever only needing to get the type ahead of time.